data-inspector first draft (#2629)
cap-js/docs: plugins/index.md

The `@cap-js/process` plugin lets you interact with the Workflow API of SAP Build Process Automation. It provides annotations and a programmatic API for working with processes. Use it to manage the lifecycle of processes — for example, to start, cancel, suspend, or resume them — and to retrieve information about running and finished processes.

[![Node.js](/logos/nodejs.svg 'Link to the plugin page.'){style="height:2.5em; display:inline; margin:0 0.2em;"}](https://github.com/cap-js/process)

> ## Data Inspector {#data-inspector-plugin}

Reviewer: nit (+ to align with release notes): 
```suggestion
## Data Inspector {#data-inspector}
```

> The [Data Inspector plugin](https://www.npmjs.com/package/@cap-js/data-inspector) enables developers and support users to inspect data from database entities and service definitions within a CAP application, in both local and production environments, while ensuring a secure and compliant experience.

Reviewer: ```suggestion
The Node.js plugin `@cap-js/data-inspector` lets developers and support users inspect data from database entities and service definitions in a CAP application, in both local and production environments. Inspection happens at the CAP service level, so all built-in authentication, authorization, and audit logging mechanisms are automatically respected.
```

> The Node.js plugin `@cap-js/data-inspector` lets developers and support users inspect data from database entities and service definitions in a CAP application, in both local and production environments. Inspection happens at the CAP service level, so all built-in authentication, authorization, and audit logging mechanisms are automatically respected.

Reviewer 2: Not sure if that distinction is really needed. We never address support users separately. They use whatever developers use. Or?

```suggestion
The Node.js plugin `@cap-js/data-inspector` allows you to inspect data from database entities and service definitions in a CAP application in local and production environments. Inspection occurs at the CAP service level. This means all built-in authentication, authorization, and audit logging mechanisms are automatically respected.
```
